src/GovUk.Education.ExploreEducationStatistics.Data.Services/SubjectMetaService.cs

```diff
@@ -106,12 +106,12 @@ private SubjectMetaViewModel GetSubjectMetaViewModelFromQuery(Subject subject, S
             var stopwatch = Stopwatch.StartNew();
             stopwatch.Start();
 
-            var filters = GetFilters(observations);
+            var filters = query.TimePeriod != null ? GetFilters(subject.Id, observations, false) : new Dictionary<string, FilterMetaViewModel>();
```

Comment:
Time periods are also not needed until we have a non-empty `LocationQuery` in the query.

I think if we make this change permanent then we could come up with a way to specify in the request what components of the meta data to return.  Someone might assume that just given a query.SubjectId or a query.SubjectId and a query.Location that it should return everything, but it will be missing filters and indicators and it's not too obvious why.

Also I'm not sure at what point this method got given a `Subject` as a parameter, becuase the SubjectId is already in the query object.

Reply:
It never passes through this code without a location - changed to use Subject id from query though.